vendor/engines/secure_rooms/app/models/secure_rooms/card_reader.rb

```diff
@@ -9,6 +9,26 @@ class CardReader < ActiveRecord::Base
 
     delegate :facility, to: :secure_room
 
+    alias_attribute :ingress, :direction_in
+
+    def egress?
+      !ingress?
```

Comment:
I initially was just going to call it `in`, but `in?` conflict's with `x.in?(array)`.